app/scripts/views/settings/two_step_authentication.js

```diff
@@ -66,6 +67,14 @@ const View = FormView.extend({
     if (this.broker.hasCapability('showTwoStepAuthentication')) {
       return true;
     }
+
+    // Two-step authentication is also enabled for all Mozilla users
+    const account = this.getSignedInAccount();
```

Comment:
@vbudhram - What are next steps past this, to open the feature gradually as an experiment? If so, should we add the experiment logic, i.e., a new grouping rule, to make it so? The advantage is you'll start to get some experiment logging.

Reply:
Yea, next steps were train 112 10% users, train 113 100%. I'll save my future self some work setup the experiment logic this train :)